### PR TITLE
 Add GetActionsByAddress() endpoint

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -544,34 +550,50 @@ func (api *Server) getActions(start uint64, count uint64) (*iotexapi.GetActionsR
...
-				return &iotexapi.GetActionsResponse{ActionInfo: res}, nil
+				return &iotexapi.GetActionsResponse{
+					Total:      totalActions,
+					ActionInfo: res,
+				}, nil
 			}
 			act, err := api.convertToAction(selps[i], true)
 			if err != nil {
-				return nil, err
+				continue
```
Comment (lizhefeng) [inline api/api.go]: Why continue?

File: api/api.go [hunk 2]
```diff
@@ -544,34 +550,50 @@ func (api *Server) getActions(start uint64, count uint64) (*iotexapi.GetActionsR
...
+	for height := uint64(1); height <= api.bc.TipHeight(); height++ {
+		blk, err := api.bc.GetBlockByHeight(height)
 		if err != nil {
 			return nil, status.Error(codes.NotFound, err.Error())
 		}
 		selps := blk.Actions
+		if !hit && start >= uint64(len(selps)) {
+			start -= uint64(len(selps))
+			continue
+		}
```
Comment (lizhefeng) [inline api/api.go]: Shouldn't we set hit to true before the for loop?

File: api/api.go (hunk 1 above)
Comment (dustinxie) [reply api/api.go]: as discussed, this is trade-off of robustness vs. accuracy

File: api/api.go (hunk 2 above)
Comment (dustinxie) [reply api/api.go]: since start > len(selps), hit remains false before the for loop below

**Pull request merged**